test/integration/generated/body-integer/Source/IntOperation.swift

```diff
@@ -614,7 +614,7 @@ public final class IntOperation {
 
     /// Put max int32 value
     /// - Parameters:
-    ///    - max32 : int body
+    ///    - max32 :
```

Comment:
why the comment 'int bodoy' is removed?

Reply:
This is something I'm looking into. I'm still trying to find where it came from in the Swagger.